installation.md

```diff
@@ -30,9 +30,17 @@ dep self-update --upgrade
 To install Deployer with Composer, run next command:
 
 ```sh
-composer require deployer/deployer
+composer require deployer/deployer:~4.0 --dev
```

Comment:
Hmmm, agree. But please remove 4.0 prefix, there is change what major version will be changing and this place will be forgoten.